### PR TITLE
Changes during review

### PR DESCRIPTION
**Pull request by marthagenaars**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,415 +32,450 @@
...
+<my-custom-element unresolved></my-custom-element>
 ```
 
 ## <a name='Custombuilt-inelementscurrentlyunsupported'></a>Custom built-in elements (currently unsupported)
-* An element extending a built in element is also known as a *customized built-in element*.
-* It is intended to work exactly like an autonomous custom element, however inheriting all behavior that the element that is extended would have.
-* In stead of extending `HTMLElement`, to create a customized built-in element it must extend a specific subclass like `HTMLButtonElement`, and supply an object specifying what element it extends as multiple elements use the same interface classes.
+* An element extending a built-in element is known as a *customized built-in element*.
+* It is intended to work exactly like an autonomous custom element, but inheriting all behavior from its parent built-in element.
+* Rather than extending `HTMLElement`, a customized built-in element it should extend one of its subclasses (like `HTMLButtonElement`), and supply a configuration object to specify what element is extended (since multiple elements can use the same interface class).
```
Comment (joppekroon) [inline README.md]: Maybe rephrase "it should extend" to "extends" and fix the rest of the sentence accordingly

File: README.md [hunk 2]
```diff
@@ -32,415 +32,450 @@
...
 const caption = document.createElement('h1');
 
 caption.appendChild(document.createTextNode('Hello Shadow DOM'));
-shadowRoot.appendChild(caption)
+shadowRoot.appendChild(caption);
 ```
 
-* A shadow tree can have a style element and even a stylesheet. These styles are local to the shadow DOM and will not bleed into the document.
+[//]: # (FIXME: What do you mean with a stylesheet, as opposed to a style element, here?)
+* A shadow tree can have a style element and even a stylesheet.
```
Comment (joppekroon) [inline README.md]: I mean a <link rel="stylesheet"> tag

Comment (joppekroon) [reply README.md]: ...A link tag to an external stylesheet

File: README.md [hunk 3]
```diff
@@ -32,415 +32,450 @@
...
+  These styles are local to the shadow DOM and will not bleed into the document.
 
 ### <a name='Slotting'></a>Slotting
+[//]: # (FIXME: Isn't basically everything in the regular HTML part of the Light DOM?)
```
Comment (joppekroon) [inline README.md]: Only the DOM enclosed in the open and close tag of the custom element is light DOM

File: README.md [hunk 4]
```diff
@@ -32,415 +32,450 @@
...
+[//]: # (FIXME: You are using "shadow tree" versus "Light DOM", y u no consistent?)
```
Comment (joppekroon) [inline README.md]: I was avoiding the words shadow DOM, but that seems dumb now

File: README.md [hunk 5]
```diff
@@ -32,415 +32,450 @@
...
+[//]: # (FIXME: The bits about "default content" aren't very clear...)
```
Comment (joppekroon) [inline README.md]: It's the DOM within a slot, which will be overwritten of the Light DOM provides content for that slot

File: README.md [hunk 6]
```diff
@@ -32,415 +32,450 @@
...
 * An autonomous custom element applies the defined behaviour to a custom tag.
 
-	```html
-	<x-element></x-element>
-	```
+```html
+<my-custom-element></my-custom-element>
+```
 
 ### <a name='Constructor'></a>Constructor
```
Comment (joppekroon) [inline README.md]: Why no quotes around this one?

File: README.md [hunk 7]
```diff
@@ -32,415 +32,450 @@
...
 
 #### <a name='FromLighttoShadow'></a>From Light to Shadow
-* The shadow DOM can exert limited influence on the host element (the element to which the shadow tree is attached) and the light DOM that is distributed into it's slots. However, in all cases, styling applied in the surrounding document will take precedence.
-* The `:host` selector selects the host element
-* The `:host(<selector>)` selector selects the host element only if the given `<selector>` matches the host.
-* The `:host-context(<compound-selector>)` selects the host element only if the given `<compound-selector>` matches the host or any of it's ancestors. This, for example, allows for theming across the page by toggling a class on the body element. (Although Custom CSS properties may be preferable here)
-* The `::slotted(<compound-selector>)` pseudo selector selects any _top level_ element from the light DOM that matches the `<compound-selector>` and is distributed into any of the shadow DOM's slots. 
+* The shadow DOM can exert limited influence on the host element (the element to which the shadow tree is attached) and the light DOM that is distributed into its slots.
```
Comment (joppekroon) [inline README.md]: Let's uppercase Light DOM consistently

File: README.md [hunk 8]
```diff
@@ -32,415 +32,450 @@
...
-	link.href = 'file.html';
-	document.head.appendChild(link);
+const link = document.createElement('link');
+link.rel = 'import';
+link.href = 'file.html';
+document.head.appendChild(link);
 ```
 
-* An HTML import will not just add its content to the document, it will make it available for use later.
+* An HTML import will not just add its content to the document, it will make it available for later use, as well.
```
Comment (joppekroon) [inline README.md]: The contents are not added at all, they are _only_ available for later use. I suggest removing "just" and "as well", and adding a "but".

File: README.md [hunk 9]
```diff
@@ -32,415 +32,450 @@
...
   This works by using `document.currentScript.ownerDocument.baseUri` to find the location of the current component, as opposed to the location of the main document where the component is imported.
 
 
 ## <a name='HTMLTemplate'></a>HTML Template
 * HTML template is the first of the WebComponent specs to be implemented by all browser vendors, only IE11 requires a polyfill.
-* A HTML template is indicated with the `<template>` tag, and can be placed virtually anywhere.
-* The content of the `<template>` tag is put into a different html document fragment by the parser and as such is completely inert. This means script won't run, markup won't render, and styles won't apply. It does not even have to be valid HTML.
-* The `content` property of a template contains the template document fragment and can be imported or cloned. Adding the clone to the DOM will activate the contents of that template, and at that time it has to be valid HTML.
+* A HTML template is indicated with the `<template>` tag, and can be placed virtually anywhere in the DOM.
```
Comment (joppekroon) [inline README.md]: I guess this should have been "an HTML"

File: README.md (hunk 6 above)
Comment (marthagenaars) [reply README.md]: Assuming you mean "Constructor": it is because this is the _word_ constructor, and not the _keyword_ `constructor`. You could use the keyword, but then it should not be capitalized.

Comment (joppekroon) [reply README.md]: I intended to enumerate the elements of a webcomponent, so yes it is intended to be the keyword and it shouldn't be capitalized 😁
